src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -461,10 +461,19 @@ class HttpGoogleServicesDAO(implicit val system: ActorSystem, implicit val mater
   }
 
   /** Fetch the latest price list from Google. Returns only the subset of prices that we find we have use for. */
-  def fetchPriceList(implicit executionContext: ExecutionContext): Future[GooglePriceList] = {
-    val httpReq = Get(FireCloudConfig.GoogleCloud.priceListUrl)
-
-    unAuthedRequestToObject[GooglePriceList](httpReq)
+  //Why is this a val? Because the price lists do not change very often. This prevents making an HTTP call to Google
+  //every time we want to calculate a cost estimate (which happens extremely often in the Terra UI)
+  //Because the price list is brittle and Google sometimes changes the names of keys in the JSON, there is a
+  //default cached value in configuration to use as a backup. If we fallback to it, the error will be logged
+  //but users will probably not notice a difference. They're cost *estimates*, after all.
+  val fetchPriceList: Future[GooglePriceList] = {
+    val httpReq = Get(priceListUrl)
+
+    unAuthedRequestToObject[GooglePriceList](httpReq).recover {
+      case t: Throwable =>
+        logger.warn(s"Unable to fetch/parse latest Google price list. A cached (possibly outdated) value will be used instead. Error: ${t.getMessage}")
```

Comment:
nit: make this an error, to ensure we get notified via Sentry or otherwise about failures